packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -407,7 +412,7 @@ const verifyRules = (parsedRules: RulesParameters, proposal: SnapshotProposalGra
     return { verified: false, error: `Proposal did not meet rules quorum of ${parsedRules.quorum}` };
 
   // Check rules voting period.
-  if ((proposal.end - proposal.start) * 3600 < parsedRules.votingPeriod)
+  if (proposal.end - proposal.start < parsedRules.votingPeriod * 3600)
```

Comment:
There was a bug in the verification logic. Vote period in rules is in hours, thus need multiply by 3600